### PR TITLE
DEVDOCS-3719 Add channel_id to the payload fields reference

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-3719](https://jira.bigcommerce.com/browse/DEVDOCS-3719)

## What changed?
Add "channel_id" to the Payload Fields reference on this page: 
Used the following description: "Optional field containing the channel_id corresponding to a headless storefront."

### CODE REVIEW

**Comment by slsriehl** [comment]
@traci707 @hatertron3000 I clarified a bit based on my understanding of MSF concepts. 

There's also a spin-off branch with some scope creep.

File: docs/api-docs/customers/customer-login-api.md [hunk 1]
```diff
@@ -19,21 +19,22 @@ You can use the Customer Login API in the following use cases:
...
+|:-----------|:-----|:------------|
+| `iss` | string | Indicates the token's issuer. This is your API account's client ID.|
+| `iat` | integer| The time when the token was generated. This is a numeric value indicating the number of seconds since the [Unix epoch](http://en.wikipedia.org/wiki/Unix_time).|
 | `jti` | string | A unique request ID (ex. uuid).|
 | `operation` | string | Must contain the string `"customer_login"`.|
-| `store_hash` | string | Store hash identifying the store you are logging into.|
-| `customer_id` | integer | ID of the customer you are logging in.|
-| `redirect_to` | string | Optional field containing a relative path for the shopper's destination after login. Will default to `/account.php`. |
+| `store_hash` | string | The store hash identifying the store the shopper is signing in to.|
+| `channel_id` | integer | Optional field containing the `channel_id` corresponding to the storefront or other sales channel the shopper is signing in to. |
```
Comment (hatertron3000) [inline docs/api-docs/customers/customer-login-api.md:35]: Other [channels](https://developer.bigcommerce.com/api-reference/b3A6MzU5MDQ0NDc-create-a-channel) may include Marketplace, POS, "marketing" (I'm not clear on the intended use her) or "other". I'm not sure how a customer would use the Customer Login API to auth against any non-storefront channels.

Consider removing "or other sales channel" from the description.

Comment (slsriehl) [reply docs/api-docs/customers/customer-login-api.md]: @hatertron3000 @traci707 My thought is something like identifying a user who's a repeat customer purchasing on a social media channel. So they're an existing customer on the store's instagram channel. The implementation that's using the Customer Login API spins up in response to webhook events.  Hazy on details from there... but it seems like this is the kind of thing for which MSF / multichannel is trying to leave open the space of possibility.

**Review by slsriehl — APPROVED** [review]
I still favor `or other sales channel` after talking with @hatertron3000 , but I'm gonna leave this in the interest of putting our energies elsewhere.

**Pull request merged**